input/jgr_grid.cfg

```diff
@@ -60,14 +60,14 @@ dt_initial                  = 8e2    # Inital step size
 # Flux convergence scheme and tolerances for surface equilibration
 flux_convergence            = 0         # 0: off | 1: on | 2: restart
 F_atm_bc                    = 0         # Boundary condition choice for F_atm, 0: TOA | 1: Surface
-F_crit                      = 0.1       # Critical flux, below which attempt to stabilise the evolution (0: disabled)
+F_crit                      = 0.0      # Critical flux, below which attempt to stabilise the evolution (0: disabled)
```

Comment:
Not necessary anymore?

Reply:
With the new time-stepping changes, I've found that this isn't needed. The step size is constrained by the relative change in the flux (and melt fraction), so if the flux is small then only small changes will be allowed to the flux anyway. Probably worth leaving this parameter in for the future, since it might be helpful later on.